lib/fileUpload.coffee

```diff
@@ -39,7 +41,8 @@ if UploadFS?
 			onFinishUpload: ->
 				console.log arguments
 			onRead: (fileId, file, req, res) ->
-				res.setHeader 'content-disposition', 'download'
+				unless file?.type?.match(/^image\/.*/)
```

Comment:
The problem with this though is if we want to see the image bigger and click it we are then forced to go to our download folders and open the file.  Seems like bad UX.  I know the few times i've done it i've been a little frustrated.  

Also isn't the route cause here the svg's?  If there is a security risk at them opening svgs, and we allow them to preview anyways?  Shouldn't we be checking this instead of just shutting off opening by direct link?